src/components/MapLegend.vue

```diff
@@ -128,16 +90,24 @@ export default {
   },
   data() {
     return {
-      legend: null
+      legend: null,
+      hidden: false,
+      infoShowing: false
     };
   },
   mounted() {
     this.createLegend();
   },
   methods: {
-      runGoogleAnalytics(eventName, action, label) {
-          this.$ga.event(eventName, action, label)
-      },
+    runGoogleAnalytics(eventName, action, label) {
+      this.$ga.event(eventName, action, label);
+    },
+    legendToggle(){
+      this.hidden = !this.hidden;
+    },
+    modalToggle(){
+      this.infoShowing = !this.infoShowing;
```

Comment:
here is a new fun way of doing these toggle -- the ternary statement
modalToggle() {
    this.infoShowing === false ? this.infoShowing = true : this.infoShowing = false
}
Basically it says, if this is false make it true, if it is true make it false. Really handy in combo with v-show.